### ISSUE TITLE
Invalid resource theme://

### ISSUE THREAD

**Issue by MorphBonehunter** [getgrav/grav-plugin-email#128]
I've run 2 (very) small websites with grav and today i tried to enable/configure the email plugin in the admin panel on both of the installations.
But every time i try to test the Plugin i got the following error (i obfuscate the settings a little bit):
```
bin/plugin email test-email -t <name>@<domain>.de

Current Configuration:

array:16 [
  "enabled" => true
  "from" => "<name>@<domain>.net"
  "from_name" => null
  "to" => "<name>@<domain>.com"
  "to_name" => null
  "queue" => array:4 [
    "enabled" => false
    "flush_frequency" => "* * * * *"
    "flush_msg_limit" => 10
    "flush_time_limit" => 100
  ]
  "mailer" => array:3 [
    "engine" => "smtp"
    "smtp" => array:5 [
      "server" => "<name>.<domain>.net"
      "port" => 25
      "encryption" => "none"
      "user" => null
      "password" => null
    ]
    "sendmail" => array:1 [
      "bin" => "/usr/sbin/sendmail -bs"
    ]
  ]
  "content_type" => "text/html"
  "debug" => false
  "charset" => null
  "cc" => null
  "cc_name" => null
  "bcc" => null
  "reply_to" => null
  "reply_to_name" => null
  "body" => null
]


In UniformResourceLocator.php line 435:
                             
  Invalid resource theme://  
                             

test-email [-t|--to TO] [-e|--env [ENV]] [-s|--subject [SUBJECT]] [-b|--body [BODY]]
```
Both sites use the Quark theme and i have no clue what the problem could be.

**Comment by mahagr** [comment]
Are you using Grav 1.6?

**Comment by MorphBonehunter** [comment]
Yes, the version is 1.6.21.

**Comment by ElGecko76** [comment]
I have same problem, @MorphBonehunter Have you found a solution?

**Comment by pawelandrzejewski89** [comment]
Same issue for Grav 1.6.21 and 'Static site generator' plugin.

**Comment by ElGecko76** [comment]
How can the problem be corrected?

**Comment by rhukster** [comment]
I’ll take a look at this today.

**Comment by randoum** [comment]
Exact same problem here

**Comment by pawelandrzejewski89** [comment]
In case of a different plugin with same error: 'Static site generator' on Grav 1.6.21 there was easy workaround -> downgrade to 1.6.20.
In Grav 1.6.21 there's some inconsistency in documentation/vendors.

Can you confirm that downgrade to 1.6.20 also work in this case and its not plugin issue?

**Comment by randoum** [comment]
Grav 1.6.21 - Email 3.0.6 (tried 3.0.5, same result)

Backtrace : 
```
In UniformResourceLocator.php line 435:
                              
  [InvalidArgumentException]  
  Invalid resource theme://   
                              

Exception trace:
 () at /vendor/rockettheme/toolbox/ResourceLocator/src/UniformResourceLocator.php:435
 RocketTheme\Toolbox\ResourceLocator\UniformResourceLocator->find() at /vendor/rockettheme/toolbox/ResourceLocator/src/UniformResourceLocator.php:403
 RocketTheme\Toolbox\ResourceLocator\UniformResourceLocator->findCached() at /vendor/rockettheme/toolbox/ResourceLocator/src/UniformResourceLocator.php:290
 RocketTheme\Toolbox\ResourceLocator\UniformResourceLocator->findResource() at /system/src/Grav/Common/Twig/Twig.php:91
 Grav\Common\Twig\Twig->init() at /user/plugins/email/classes/Utils.php:33
 Grav\Plugin\Email\Utils::sendEmail() at /user/plugins/email/cli/TestEmailCommand.php:91
 Grav\Plugin\Console\TestEmailCommand->serve() at /system/src/Grav/Console/ConsoleCommand.php:41
 Grav\Console\ConsoleCommand->execute() at /vendor/symfony/console/Command/Command.php:255
 Symfony\Component\Console\Command\Command->run() at /vendor/symfony/console/Application.php:917
 Symfony\Component\Console\Application->doRunCommand() at /vendor/symfony/console/Application.php:269
 Symfony\Component\Console\Application->doRun() at /vendor/symfony/console/Application.php:145
 Symfony\Component\Console\Application->run() at /bin/plugin:152
```

**Comment by mahagr** [comment]
The issue is caused by this change https://github.com/getgrav/grav/blob/1.7/UPGRADE-1.7.md#cli-1

**Comment by mahagr** [comment]
Here is the change itself https://github.com/getgrav/grav/commit/48170d2fa01cdeab8d60d0dd6cd563ba9316970b

**Comment by mahagr** [comment]
So.. Basically the reason why this (and some other plugins) CLI commands broke is because of Grav initialization was broken in older versions and they were initializing theme too early, causing issues which could not be fixed without breaking some of the CLI commands.

**Comment by MorphBonehunter** [comment]
Confirm that the issue is fixed with 3.0.7:
`Message sent successfully!`
